routes/index.js

```diff
@@ -5,7 +5,6 @@ var crypter = require("../util/crypter.js")
 var validation = require("../public/scripts/validation.js");
 /* GET home page. */
 router.get("/", function(req, res, next) {
-  exports.hostname = req.get('host');
```

Comment:
At the end of this file module.exports is being set to router. Does that override `exports.hostname`? not sure about this.